README.md

```diff
@@ -5,14 +5,15 @@ QJSLib is a JavaScript utility library that provides functionality for QRadar ap
 * [CONTRIBUTING](CONTRIBUTING.md)
 * [MAINTAINERS](MAINTAINERS.md)
 * [CHANGELOG](CHANGELOG.md)
+* [DOCUMENTATION](https://github.com/IBM/qjslib/wiki/qappfw)
 
 ## How to use
 ### Browser vs module
-The build process for this project generates two seperate *qappfw.js* files, one in *build/* and one in *lib/*. 
+The build process for this project generates two seperate files in *lib/*.
 
-The file in *lib/* is not minified and designed to be bundled with your application code.
+The file in *qappfw.js* is not minified and designed to be bundled with your application code.
```

Comment:
Bit of a weird sentence

Reply:
*qappfw.js* is not minified, it is designed to be bundled with your application code.

Does this read a bit better?

Reply:
qappfw.js is not minified. It is designed to be bundled with your application code.